index.html

```diff
@@ -63,6 +63,41 @@ <h2>Currency Converter</h2>
         </form>
       </section>
 
+      <section id="tabel-container">
+        <h2>Currencies Table</h2>
+        <table class="table">
+          <tr id="table-currency-names-row">
+            <th>Currency Name:</th>
+          </tr>
+          <tr id="table-listing-date-row">
+            <th>Listing Date:</th>
+          </tr>
+          <tr id="table-available-rates-row">
+            <th>Available Rates:</th>
+          </tr>
+        </table>       
+      </section>
+
+      <section>
+        <h2>Search For A Specific Exchange Rate</h2>
+        <div>
+          <label for="from-currency-input">From</label>
+          <input type="text" id="from-currency-input" placeholder="Enter Currency">
+        </div>
+        <div>
+          <img src="./change.png" class="change-img"  id="change-img">
+        </div>
```

Comment:
This should be a button tag for better semantics and accessibility. For example, right now you are not able to access it through keyboard navigation with Tab key.
```suggestion
        <button type="button" class="change-currency__button">
          <img src="./change.png" class="change-img"  id="change-img">
        </button>
```